layers/meta-balena-allwinner/recipes-kernel/linux/linux-4.19/0000-0001-clk-sunxi-ng-Set-maximum-M-1-for-H3-pll-cpux-clock.patch

```diff
@@ -0,0 +1,55 @@
+From 5b367397da1947fcbf6ed1091c351808218e59bc Mon Sep 17 00:00:00 2001
+From: Ondrej Jirman <megous@megous.com>
+Date: Thu, 12 Jan 2017 16:34:57 +0100
+Subject: [PATCH 01/82] clk: sunxi-ng: Set maximum M = 1 for H3 pll-cpux clock
+
+When using M factor greater than 1 system is experiencing
+occasional lockups.
+
+This change was verified to fix lockups with PLL stress
+tester available at https://github.com/megous/h3-firmware.
+
+Note that M factor must not be used outside the kernel
+either, so for example u-boot needs a similar patch.
+---
```

Comment:
These patches are missing `Signed-off-by` and `Upstream-status`.

Reply:
The patches were ported from Armbian. They are about 360. I don't think it's feasible to add a Singed-off-by and Upstream-status in all of them

Reply:
:) ok then, let's leave them like that but please mention the revision (sha) of the armbiam git repo from where you imported them.